src/odemis/driver/xt_client.py

```diff
@@ -1099,133 +1065,125 @@ def get_dc_coils(self):
             self.server._pyroClaimOwnership()
             return self.server.get_dc_coils()
 
-    def get_use_case(self):
+    def get_use_case(self) -> str:
         """
         Get the current use case state. The use case reflects whether the system
         is currently in multi-beam or single beam mode.
 
-        Returns
-        -------
-        state: str, 'MultiBeamTile' or 'SingleBeamlet'
-
+        :return: 'MultiBeamTile' or 'SingleBeamlet'
         """
         with self._proxy_access:
             self.server._pyroClaimOwnership()
             return self.server.get_use_case()
 
-    def set_use_case(self, state):
+    def set_use_case(self, state: str) -> None:
         """
         Set the current use case state. The use case reflects whether the system
         is currently in multi-beam or single beam mode.
 
-        Parameters
-        ----------
-        state: str, 'MultiBeamTile' or 'SingleBeamlet'
-
+        :param state: 'MultiBeamTile' or 'SingleBeamlet'
         """
         with self._proxy_access:
             self.server._pyroClaimOwnership()
-            return self.server.set_use_case(state)
+            self.server.set_use_case(state)
 
-    def get_mpp_orientation(self):
+    def get_mpp_orientation(self) -> float:
         """
         Get the current multi probe pattern orientation in degrees.
 
-        :return (float): Multi probe orientation in degrees
+        :return: Multi probe orientation in degrees
         """
         with self._proxy_access:
             self.server._pyroClaimOwnership()
             return self.server.get_mpp_orientation()
 
-    def mpp_orientation_info(self):
+    def mpp_orientation_info(self) -> Dict[str, list]:
```

Comment:
Is that always a `list` for values? The docstring seems to imply it can be `'degrees'`. So maybe just write `Any`?